llvm/lib/IR/LLVMContextImpl.h

```diff
@@ -1520,9 +1520,10 @@ class LLVMContextImpl {
   ConstantInt *TheFalseVal = nullptr;
 
   // Basic type instances.
-  Type VoidTy, LabelTy, HalfTy, BFloatTy, FloatTy, DoubleTy, MetadataTy,
+  Type VoidTy, LabelTy, HalfTy, BFloatTy, FloatTy, Decimal32Ty, DoubleTy,
```

Comment:
I have no idea why the Type variables here are declared with two different expressions, but this looks wrong (before and after your change). It looks like the two different lines were used when the type system was rewritten 12 years ago. Probably as a formatting thing to avoid having variables on a different line than their type. The TokenTy got bumped to a second line when BFloatTy was added. I'd probably reformat this whole thing into three lines.

I am aware that I'm being really picky at this point.

Reply:
Is that what you mean?

Reply:
Yes, what you did is more or less what I had in mind.